### DISCUSSION TITLE
Can flux make use of Tailwind merge?

### DISCUSSION THREAD

**Discussion by MartijnCuppens** [livewire/flux#315] [Feature Requests]
Instead of overriding tailwind classes with `!` modifiers, could flux make use of [Tailwind merge](https://github.com/gehrisandro/tailwind-merge-laravel)?

The idea is that it replaces classes with the same meaning:

```html
// circle.blade.php
<div {{ $attributes->twMerge('w-10 h-10 rounded-full bg-red-500') }}></div>

// your-view.blade.php
<x-circle class="bg-blue-500" />

// output
<div class="w-10 h-10 rounded-full bg-blue-500"></div>
```

**Comment by daugaard47** [comment]
This ^

**Comment by Arkhas** [comment]
any news about this ?

**Reply by joshhanley** [reply]
@Arkhas Caleb had said in one of his podcasts when he was releasing Flux that he won't be using Tailwind merge.
